bedwars-plugin/src/main/java/com/andrei1058/bedwars/listeners/BreakPlace.java

```diff
@@ -161,6 +161,10 @@ public void onBlockPlace(BlockPlaceEvent e) {
                 e.setCancelled(true);
                 return;
             }
+            if (e.getBlockPlaced().getLocation().getBlockY() <= a.getConfig().getInt(ConfigPath.ARENA_CONFIGURATION_MIN_BUILD_Y)) {
+                e.setCancelled(true);
```

Comment:
Would you also like to send a message that the min height has been reached? :)

Reply:
Done (sorry for the delayed response)